web/libs/editor/src/tags/object/Video/Video.js

```diff
@@ -197,13 +207,87 @@ const Model = types
         }
       },
 
-      addRegion(data) {
-        const control = self.videoControl() ?? self.control();
+      /**
+       * Update per frame classifications view; analogue for `updateFromResult()` but with data for current frame
+       */
+      updatePerFrameViews() {
```

Comment:
there is alot going on here - would be great to have a test of some sort to check this going forward